locale/en/blog/announcements/interactive-2016-north-america-schedule.md

```diff
@@ -81,20 +81,20 @@ Experts from the leading open source projects and enterprises will share their e
 
 - “Games as Conversational Interfaces” - Kevin Zurawel of Braintree
 - “Agile Security for Web Developers” - Kim Carter of BinaryMist
-- “Science Meets Industry: Online Behavioral Experiments with nodeGame” - Stefano Balietti from Northeastern University
+- “Science Meets Industry: Online Behavioral Experiments with nodeGame” - Stefano Balietti of Northeastern University
 - “Building Desktop Applications With Node.js Using Electron” - Steve Kinney of Turing School of Software and Design
 
 Free onsite childcare for attendees is available as well as ASL, interpretation and transcription assistance upon request. Please email events@node.js for more information.
 
-Node.js Interactive is made possible by support from Platinum Sponsor IBM and Google Cloud Platform; Gold Sponsors nearForm and NodeSource; Silver Sponsor GoDaddy, Langa, Opbeat, Rollbar, and Sauce Labs; and Bronze Sponsors Codiscope, Sqreen, and Stormpath. If you are interested in sponsoring please contact Todd Benzies at tbenzies@linuxfoundation.org.
+Node.js Interactive is made possible by support from Platinum Sponsors IBM and Google Cloud Platform; Gold Sponsors nearForm and NodeSource; Silver Sponsors GoDaddy, Langa, Opbeat, Rollbar, and Sauce Labs; and Bronze Sponsors Codiscope, Sqreen, and Stormpath. If you are interested in sponsoring please contact Todd Benzies at tbenzies@linuxfoundation.org.
 
-For attendees who register before November 14, the standard registration fee is $600; registration increases to $800 after November 14. Visit here to register.
+For attendees who register before November 14, the standard registration fee is $600; registration increases to $800 after November 14. Visit [here](http://events.linuxfoundation.org/events/node-interactive) to register.
```

Comment:
i think this should be https://www.regonline.com/nodejsinteractive2016us, shouldn't it?